src/Nethermind/Nethermind.Network.Test/ForkInfoTests.cs

```diff
@@ -159,8 +159,9 @@ public void Fork_id_and_hash_as_expected_on_gnosis(long head, ulong headTimestam
     }
 
     [TestCase(0L, 0UL, "0x50d39d7b", ChiadoSpecProvider.ShanghaiTimestamp, "Chiado genesis")]
-    [TestCase(3945317, ChiadoSpecProvider.ShanghaiTimestamp, "0xa15a4252", 0ul, "First Shanghai timestamp")]
-    [TestCase(9945337, 9984930320ul, "0xa15a4252", 0ul, "Future Shanghai timestamp")]
+    [TestCase(3945317, ChiadoSpecProvider.ShanghaiTimestamp, "0xa15a4252", ChiadoSpecProvider.CancunTimestamp, "First Shanghai timestamp")]
+    [TestCase(4_000_000, ChiadoSpecProvider.CancunTimestamp, "0x5fbc16bc", 0ul, "First Cancun timestamp")]
```

Comment:
Can you confirm fork hash with Erigon?

Reply:
Not now. I'm gonna do that after the timestamp is confirmed.

Reply:
Confirmed